src/main/java/com/GeorgesServer/app/App.java

```diff
@@ -1,8 +1,6 @@
 package com.GeorgesServer.app;
 
 
-import com.GeorgesServer.app.com.GeorgesServer.response.HttpResponseBuilder;
```

Comment:
If you're not already using one, good linter can give you a heads up on unused imports, vars, etc